### ISSUE TITLE
Login function

### ISSUE THREAD

**Issue by whatisgravity** [go-steem/rpc#1]
The login function is missing from the API. It seems relatively easy to add but I'm not sure about where you would prefer it within the structure of the library. If you have any recommendations please notify me and I will add it and issue a pull request.

**Comment by whatisgravity** [comment]
I understand I can just use the call function and do this within my own software if you feel this is out of scope but I thought it might be easy for developers if there were some basic call functions ready to use.

**Comment by whatisgravity** [comment]
I decided to make a new folder under apis called wallet I'm modeling it after database structure, so three files:
- api.go
- constants.go
- data.go

Api.go will contain all the calls related to building transactions etc

If you have any problems or suggestions let me know and I can adjust it, once it is ready I will issue a pull request for code review.

Edit: Correction, looking over the steem code, seems we need network_broadcast and login folders. Still investigating though.

**Comment by tchap** [comment]
Yeah, only `database_api` is supported now.

**Comment by tchap** [comment]
What you are talking about is `login_api` I guess.

**Comment by tchap** [comment]
We would need to implement `apis/login` and make it available as `Client.Login`.

**Comment by whatisgravity** [comment]
I have no interest in working on this, steem is not a decentralized platform. They do not care about cenorship. 

Steem is a farse to get money while using decentralization as a marketing tool. You guys should be ahsamed of yourselves.

They changed the code of the client, suddenly had 54% of the network and forked then those nodes disappeared. They use their **closed source** platform to censor things that exist on the blockchain. I did analysis and several posts have been altered from what they were on the blockchain, and not for security reasons but for political reasons.

**Comment by tchap** [comment]
@whatisgravity I am not a part of the team, so... Sure, there are some issue right now, the whole thing is not as decentralised as it could be. But not sure what you are talking about right now. Is it the fork following the hack that happened? What posts exactly? Weren't they the infected ones?
